### PR TITLE
Add options to MakeUserCommand

### PR DESCRIPTION
**Pull request by DenisaHalmaghi**
Hello 👋 ,

Thank you for writing this amazing package.

This pull request adds optional command line parameters to facilitate command line interaction in scripts.

Signed-off-by: DenisaHalmaghi <denisa@graffino.com>

### CODE REVIEW

File: packages/admin/src/Commands/MakeUserCommand.php [hunk 1]
```diff
@@ -17,14 +17,17 @@ class MakeUserCommand extends Command
...
+        {--email= : The email of the user}
+        {--password= : The password of the user}';
 
     protected function getUserData(): array
     {
         return [
-            'name' => $this->validateInput(fn () => $this->ask('Name'), 'name', ['required']),
-            'email' => $this->validateInput(fn () => $this->ask('Email address'), 'email', ['required', 'email', 'unique:' . $this->getUserModel()]),
-            'password' => Hash::make($this->validateInput(fn () => $this->secret('Password'), 'password', ['required', 'min:8'])),
+            'name' => $this->validateInput(fn () =>  $this->option('name') ? $this->option('name') : $this->ask('Name'), 'name', ['required']),
```
Comment (danharrin) [inline packages/admin/src/Commands/MakeUserCommand.php:28]: How about `$this->option('name') ?? $this->ask('Name')` etc

Comment (zepfietje) [reply packages/admin/src/Commands/MakeUserCommand.php:28]: Should work. Wanted to commit this change, but I couldn't push to this PR.

**Comment by zepfietje** [comment]
Something I noticed while testing this change locally: when providing invalid input using the `--email=test` option for example, the validation error output gets looped infinitely.

**Comment by danharrin** [comment]
Thats because the validation method continually checks for new input, and since the option is provided in the CLI command it keeps failing.

**Comment by danharrin** [comment]
Hey @DenisaHalmaghi, I appreciate the PR but we cannot merge it in the current state, you haven't responded, and we're unable to edit the code ourselves.

**Pull request closed without merging**